### PR TITLE
Update Android documentation for Google Play signed app domain verification

### PR DESCRIPTION
**Pull request by paulpascal**
# Description

For apps distributed through the Play Store using Google Play Console signing, domain verification fails even after adding the SHA256 fingerprint used to build the app.

This PR updates the CHT Android Flavors documentation to include detailed instructions for apps signed by Google Play. Specifically, it addresses the need to use the SHA256 fingerprint provided by the Google Play Console for successful domain verification.

# Importance

These updates are crucial for ensuring that apps signed by Google Play are correctly associated with their respective domains, allowing for seamless deep linking and improved user experience.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: content/en/apps/guides/android/branding.md [hunk 1]
```diff
@@ -257,6 +257,20 @@ All you have to do to make the CHT serve your assetlinks at `/.well-known/assetl
...
+
+#### Steps to Retrieve SHA256 Fingerprint from Google Play
+
+1. **Log in to Google Play Console**.
+2. **Navigate to Your App**: Select your app from the list of published applications.
+3. **Setup > App Signing**: In the left-hand menu, under the Setup Menu, there is App Signing.
+4. **Find SHA256 Fingerprint**: Google Play will display the SHA256 fingerprint required for your app.
+5. **Update assetlinks.json**: Use this SHA256 fingerprint in your `assetlinks.json` file.
+
+Once added and pushed to the server, the deep link can be monitored from the console as well, under the **Deep Links** in the left-hand menu where the Play Console allows you to check the deep link settings and also provides hints to fix any errors (providing the right SHA256 Fingerprint to add).
```
Comment (binokaryg) [inline content/en/apps/guides/android/branding.md:272]:
```suggestion
Once added and pushed to the server, the deep link can be monitored from the console as well, under the **Deep Links** in the left-hand menu where the Play Console allows you to check the deep link settings and also provides hints to fix any errors (providing the right SHA256 fingerprint to add).
```

File: content/en/apps/guides/android/branding.md [hunk 2]
```diff
@@ -257,6 +257,20 @@ All you have to do to make the CHT serve your assetlinks at `/.well-known/assetl
...
+#### Note for Apps Using Google Play Signing
+
+For apps signed by Google Play, you need to use the SHA256 fingerprint provided in the Google Play Console to ensure successful domain verification.
+
+#### Steps to Retrieve SHA256 Fingerprint from Google Play
+
+1. **Log in to Google Play Console**.
+2. **Navigate to Your App**: Select your app from the list of published applications.
+3. **Setup > App Signing**: In the left-hand menu, under the Setup Menu, there is App Signing.
+4. **Find SHA256 Fingerprint**: Google Play will display the SHA256 fingerprint required for your app.
```
Comment (binokaryg) [inline content/en/apps/guides/android/branding.md:269]: Play Store also provides the full JSON.

Comment (binokaryg) [inline content/en/apps/guides/android/branding.md:269]:
```suggestion
2. **Navigate to your app**: Select your app from the list of published applications.
3. **Setup > App signing**: In the left-hand menu, under the Setup Menu, there is App Signing.
4. **Find SHA256 fingerprint**: Google Play will display the SHA256 fingerprint required for your app.
```

**Review by binokaryg — APPROVED** [review]
Thanks for this added information.

I have noticed that if you select to upload your own key and use it for app signing instead of the one provided by Google, the SHA256 fingerprint should match.

It's still a good idea to verify, and since the Play store also provides full JSON content, it could be easier for the developers to copy it from there.

I have suggested a few changes in the letter case.

**Review by jkuester — APPROVED** [review]
@paulpascal this is great! Thank you for adding this context.

**Pull request merged**